ouroboros-network/src/Ouroboros/Network/Protocol/ChainSync/PipelineDecision.hs

```diff
@@ -179,7 +179,7 @@ pipelineDecisionLowHighMark lowMark highMark =
     goZero :: Nat Z -> WithOrigin BlockNo -> WithOrigin BlockNo
            -> (PipelineDecision Z, MkPipelineDecision)
     goZero Zero clientTipBlockNo serverTipBlockNo
-      | clientTipBlockNo `bnoPlus` 1 == serverTipBlockNo
```

Comment:
Caught an off-by-one error here?

Reply:
Ah, I see the separate commit message. 👍 (The PR title and description didn't lead me to expect this.)